config/localization/en/app.json

```diff
@@ -1122,7 +1123,7 @@
       "LEARN_ABOUT_WALLETS": "Learn about wallets",
       "SELECT_LANGUAGE": "Select language",
       "SELECT_WALLET_FROM_OPTIONS": "Please select your wallet from the options below.",
-      "WALLET_DEFINITION": "A wallet is a device or application that stores your keys and enables you to explore crypto."
+      "WALLET_DEFINITION": "A wallet is a device or application that stores your keys and enables you to explore crypto.\n{ABOUT_WALLETS_LINK}"
```

Comment:
Is \n best practice for these? Usually new lines are better with <p> tags or something.